sass/style.scss

```diff
@@ -210,7 +212,7 @@ a {
 h2.state {
   background: #eee;
   color: #000;
-  font: 800 24px 'Roboto';
+  font: 800 24px "Roboto";
```

Comment:
noticing nits cause of the fomatting changes. We don't use roboto anymore and I don't think we use this class either. We can do a separate styles cleanup pass

Reply:
Yeah I'll create another issue for that